impl/sql/adminstorage/admin_storage.go

```diff
@@ -36,24 +36,26 @@ CREATE TABLE IF NOT EXISTS Domains(
   LogId                 BIGINT NOT NULL,
   VRFPublicKey          MEDIUMBLOB NOT NULL,
   VRFPrivateKey         MEDIUMBLOB NOT NULL,
+  MinInterval		BIGINT NOT NULL,
```

Comment:
Column-align the SQL types

Reply:
This was a case of tabs vs. spaces. Now it's all spaces.